### DISCUSSION TITLE
Will there be support for this on VSCodium in the near future?

### DISCUSSION THREAD

**Discussion by stereotypicaldev** [illixion/vscode-vibrancy-continued#110] [Q&A] [answered]
Will there be a fork of this over on the Open Registry for users of VSCodium?

**Comment by illixion** [comment]
I was planning to set up a GitHub Action to auto-publish the extension, so I can definitely add Open Registry as a target

**Comment by illixion** [comment, marked as answer]
Vibrancy is now also published to Open VSX Registry: https://open-vsx.org/extension/illixion/vscode-vibrancy-continued

The page looks a bit wrong with the light gray header and white text, I’ll see what I can do about this later.

**Reply by stereotypicaldev** [reply]
Awesome! Thanks for both the quick reply and the publication. Keep up the good work.
